sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -2721,22 +2734,22 @@ Accept wildcard characters: False
 ```
 
 ### -EnableAutoExpirationVersionTrim
-Global and SharePoint admins in Microsoft 365 can set Organization-level Version History Limit settings that universally apply to new versions created on all new Document Libraries created in your organization. 
+Global and SharePoint Administrators in Microsoft 365 can set organization-level version history limits settings that universally apply to new versions created on all new document libraries created in your organization. 
 
-When Version History Limits are managed Automatically, SharePoint employs an algorithm behind the scenes that deletes (thins out) intermittent older versions that are least likely to be needed, while preserving sufficient high-value versions - more versions in the recent past and fewer farther back in time - in case restores are required.
+When version history limits are managed automatically, SharePoint employs an algorithm behind the scenes that deletes (thins out) intermittent older versions that are least likely to be needed, while preserving sufficient high-value versions - more versions in the recent past and fewer farther back in time - in case restores are required.
 
 The valid values are: 
 
-- True – Version History Limits for new versions created on all new Document Libraries in your organization will be managed Automatically.  
-- False – Version History Limits for new Versions created on all new Document Libraries in your organization will be managed Manually by setting limits to the number of major versions (MajorVersionLimit) and time set (ExpireVersionsAfterDays).  Review the documentation of both parameters to manage your organization's version limits Manually.  
+- True – Version history limits for new versions created on all new document libraries in your organization will be managed automatically.  
+- False – Version history limits for new Versions created on all new document libraries in your organization will be managed manually by setting limits to the number of major versions (`MajorVersionLimit`) and time set (`ExpireVersionsAfterDays`).  Review the documentation of both parameters to manage your organization's version limits manually.  
```

Comment:
>   [](http://example.com/codeflow?start=347&length=2)

remove extra space

Reply:
(I'm using CodeFlow to review the PR, and CodeFlow does a better job showing things like trailing spaces)